packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { StepDriver } from "../../interface/stepDriver";
+import { DriverContext } from "../../interface/commonArgs";
+import { UpdateAadAppArgs } from "./interface/updateAadAppArgs";
+import { Service } from "typedi";
+import { InvalidParameterUserError } from "./error/invalidParameterUserError";
+import { UpdateAadAppOutput } from "./interface/updateAadAppOutput";
+import * as fs from "fs-extra";
+import * as path from "path";
+import { AadAppClient } from "./utility/aadAppClient";
+import axios from "axios";
+import { SystemError, UserError } from "@microsoft/teamsfx-api";
+import { UnhandledSystemError, UnhandledUserError } from "./error/unhandledError";
+import { getUuid } from "../../../common/tools";
+import { expandEnvironmentVariable } from "../../utils/common";
+import { AadManifestHelper } from "../../resource/aadApp/utils/aadManifestHelper";
+import { AADManifest } from "../../resource/aadApp/interfaces/AADManifest";
+import { MissingFieldInManifestUserError } from "./error/invalidFieldInManifestError";
+
+const actionName = "aadApp/update"; // DO NOT MODIFY the name
```

Comment:
Why "DO NOT MODIFY the name", if in the future we want to refactor this name, how to do it?

Reply:
user will reference this name in teamsfx.yml, if we changed the name, it will be a breaking change.
``` yaml
provision:
  - uses: "aadApp/update"
    with:
        manifestPath: "xxx"
```